tests/acceptance/a11y-audit-test.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
 import { module, test } from 'qunit';
 import startApp from '../../tests/helpers/start-app';
-import a11yAudit from 'ember-a11y-testing/test-support/audit';
+import { a11yAudit } from 'ember-a11y-testing/test-support/audit';
```

Comment:
I would write a helper to retrieve `window.location` which you can then stub. Like so:

```js
export {
  getLocation() {
    return window && window.location;
  }
};
```
Then in the test:
```js
import helper from '../helper'; // come up with better names here
helper.getLocation = function() { retun mockLocationObject; }
```